armi/reactor/blockParameters.py

```diff
@@ -345,7 +348,10 @@ def buGroupNum(self, buGroupNum):
         pb.defParam(
             "axialExpTargetComponent",
             units=units.UNITLESS,
-            description="The name of the target component used for axial expansion and contraction of solid components.",
+            description=(
+                "The name of the target component used for axial expansion and "
+                "contraction of solid components."
```

Comment:
This edit seems odd. This line isn't over 120 chars?

Reply:
Before this PR (the version on the left) is 121 characters long.

I know 121 isn't _much_ over 120, but it's enough that `ruff` complained at me and I had to fix it.